packages/compute-baseline/src/baseline/index.ts

```diff
@@ -101,10 +106,12 @@ export function computeBaseline(
   const keystoneDate = findKeystoneDate(
     statuses.flatMap((s) => [...s.support.values()]),
   );
-  const { baseline, baseline_low_date, baseline_high_date, discouraged } =
+  const discouraged = statuses.some((s) => s.discouraged);
```

Comment:
I pulled out `discouraged`, previously it was passed into `keystoneDateToStatus` only to be returned unchanged.